simpleprophet/README.md

```diff
@@ -2,6 +2,36 @@
 
 This directory contains the simpleprophet forecast models and accompanying tools.  The simpleprophet models prioritize simplicity and stability, adding complexity and "bendiness" only when it significantly improves model performance on a holdback period.
 
+## Setup
+
+You will need a python environment with `fbprophet` and a few other
+dependencies installed. We provide a Docker image that can be pulled from GCR
+and run interactively like:
+
+```
+GOOGLE_APPLICATION_CREDENTIALS=/path/to/creds.json
+docker run -it \
+  -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/key.json \
+  -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/keys/key.json:ro \
+  --entrypoint python \
+  gcr.io/moz-fx-data-forecasting/simpleprophet
+```
+
+Or you can make code updates and build the image locally:
+
+```
+docker build . --tag simpleprophet
+docker run -it \
+  -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/key.json \
+  -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/keys/key.json:ro \
+  --entrypoint python \
+  simpleprophet
+```
+
+If you want to produce a local environment outside docker, see the
+[`fbprophet` installation instructions](https://facebook.github.io/prophet/docs/installation.html)
+and then use `pip install -r requirements.txt` to install remaining dependencies.
```

Comment:
If you're going the conda route, it may be worth considering adding an [`environment.yml`](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment), which can be used to build a virtual environment.